slice.py

```diff
@@ -36,7 +36,7 @@ def slice_clip(inFile, outDir, minSilence, threshold, verbose):
                 print("Exporting {}...".format(fileName))
                 print("\tChunk Size: {}".format(SU.ms2hms(len(chunk))))
 
-            normalized_chunk.export(outDir + fileName, format = "wav")
+            normalized_chunk.export(outDir + "/"  + fileName, format = "wav")
```

Comment:
I think the issue is if the user includes the / as part of outDir then it works, otherwise this fix is necessary. It would be better to detect if the last char of outDir is '/' and then act appropriately.
Something like " if outDir[-1] != '/': outDir += '/' "